anvio/docs/programs/anvi-script-filter-hmm-hit-table.md

```diff
@@ -0,0 +1,27 @@
+This program is for filtering a %(hmm-source)s from a %(hmm-hits)s in a %(contigs-db)s using HMM alignment parameters such as query-coverage and target-coverage. Briefly, the program will remove all records from an %(hmm-source)s in the %(hmm-hits)s then import a new %(hmm-hits)s table into the %(contigs-db)s that was filtered to your specifications. At the moment, this tool is only designed to work with `hmmsearch` with protein sequences. The `--domtblout` can be produced from running `anvi-run-hmms` with your %(hmm-source)s of interest and using the `--domtblout` parameter AND  `hmmsearch` as the program.
```

Comment:
If you don't provide `hmmsearch` as the program is an error raised? Maybe the program should automatically be set to `hmmsearch` if `--domtblout` is provided.

Reply:
That's a great point @ekiefl! For this PR, I was mainly focusing on `anvi-script-filter-hmm-hits-table` so I did not think about checks and balances for the `-domtblout ` in `anvi-run-hmms`. In fact, I added the bare minimum so that I could start developing `anvi-script-filter-hmm-hits-table`. Do you think that functionality should be addressed in this PR. If so, I would be happy to start to improving this.

Reply:
In this [commit](https://github.com/merenlab/anvio/pull/1678/commits/76a6b733e66e4363bf370c2c96956eeba1c20951) I added a config Error to stop the user from using hmmscan with `--hmm-domain-tblout-path`